src/policy/policy.cpp

```diff
@@ -132,7 +132,7 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
     TxoutType whichType;
     for (const CTxOut& txout : tx.vout) {
         if (!::IsStandard(txout.scriptPubKey, max_datacarrier_bytes, whichType)) {
-            reason = "scriptpubkey";
+            reason = "non-standard-scriptpubkey";
```

Comment:
`scriptpubkey-nonstandard` would fit the usual reject reason format better

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25791/commits/f1c810b6c7a17fa85f4eb075c061427fe59880ea.